### DISCUSSION TITLE
How can I setup a very-wide data grid?

### DISCUSSION THREAD

**Discussion by NickHirras** [microsoft/fluentui-blazor#2021] [Q&A] [answered]
I have a use case where we have a screen that needs to display more data than can fit in 100% width.  I have added an `overflow: scroll` style to the datagrid, and the user is able to resize columns to make content wider than available screen size, and then scroll horizontally.  This is working well.

How can I setup my table so that in initial view it is wider than the screen/window by default, so that user doesn't need to resize multiple columns to get the screen into a usable state?  

Screen initially will load and size columns small enough to fit in the available width.  Once user starts resizing columns, the table can grow to > 100%.  Can I set an initial width to something like `200%` or `1800px` for example?

**Comment by vnbaaij** [comment, marked as answer]
See https://fluentui-blazor.net/DataGrid#custompagingui for an example of a Datagrid that starts out wider and uses horizontal scroll

**Comment by NickHirras** [comment]
@vnbaaij that worked perfectly, thanks for the link!!
